src/sagemaker/tuner.py

```diff
@@ -525,3 +721,49 @@ def stop(self):
 
     def wait(self):
         self.sagemaker_session.wait_for_tuning_job(self.name)
+
+
+def create_identical_dataset_and_algorithm_tuner(parent, additional_parents=None, sagemaker_session=None):
+    """Creates a new tuner by copying the request fields from the provided parent to the new instance of
+        HyperarameterTuner. Followed by addition of warm start configuration with the type as
+        "IdenticalDataAndAlgorithm" and parents as the union of provided list of additional_parents and the parent.
+
+    Args:
+        parent (str): Primary parent tuning job's name from which the Tuner and Estimator configuration has to be copied
+        additional_parents (set{str}): Set of additional parent tuning job's names along with the primary parent tuning
+            job name to be used in warm starting the transfer learning tuner.
+        sagemaker_session (sagemaker.session.Session): Session object which manages interactions with
+                Amazon SageMaker APIs and any other AWS services needed. If not specified, one is created
+                using the default AWS configuration chain.
+
+    Returns:
+        sagemaker.tuner.HyperparameterTuner: New instance of warm started HyperparameterTuner
```

Comment:
nit: s/warm started/warm-started

this might sound better as something like "a new HyperparameterTuner object for the warm-started hyperparameter tuning job"?

Reply:
done